index.html

```diff
@@ -20,35 +20,38 @@
 <body data-theme="theme1">
 
   <nav class="top">
-    <a id="toplink" href="#footerlink">Ugrás a lap aljára</a>
+    <a id="toplink" href="#footerlink" aria-label="Jump to the bottom of the page">Ugrás a lap aljára</a>
```

Comment:
A weboldal magyar nyelvre van allitva lasd (lasd: `<html lang="hu"`), ezert celszerü lenne magyar tartalmat irni az aria-label-be.

Reply:
Ez ervenyes a tovabbi modositasokra is a PR-ben

Reply:
Az anchor-tagekhez ebben az esetben `role="button"` attributum indokolt lenne

Reply:
Ez is ervenyes a többi anchor-ra is a dokumentumban